politeiawww/user/localdb/localdb.go

```diff
@@ -206,6 +206,54 @@ func (l *localdb) UserGetByPubKey(pubKey string) (*user.User, error) {
 	return nil, user.ErrUserNotFound
 }
 
+func (l *localdb) UsersGetByPubKey(pubKeys []string) (map[string]user.User, error) {
```

Comment:
Exported functions require a comment in the form of:

`// UsersGetByPubKey ...`